src/parser/csql_grammar.y

```diff
@@ -5341,31 +5395,67 @@ meta_class_spec
 			  {
 			    ocs->info.spec.entity_name = $2;
 			    ocs->info.spec.only_all = PT_ONLY;
-			    ocs->info.spec.meta_class = PT_CLASS;
+			    ocs->info.spec.meta_class = PT_META_CLASS;
 			  }
 
-			if (ocs)
-			  ocs->info.spec.meta_class = PT_META_CLASS;
 			$$ = ocs;
 			PARSER_SAVE_ERR_CONTEXT ($$, @$.buffer_pos)
 
 		DBG_PRINT}}
-	;
+        | CLASS class_name_with_server_name
+                {{DBG_TRACE_GRAMMAR(meta_class_spec, : CLASS class_name_with_server_name );
+                       PT_NODE *ocs = parser_new_node (this_parser, PT_SPEC);
+			if (ocs)
+			  {
+                            ocs->info.spec.entity_name = CONTAINER_AT_0 ($2);
+			    /* for DBLink DML */
+                            ocs->info.spec.remote_server_name = CONTAINER_AT_1 ($2);
+			    ocs->info.spec.only_all = PT_ONLY;
+                            ocs->info.spec.meta_class = PT_META_CLASS;
+			  }
 
-only_all_class_spec
-	: only_class_name opt_partition_spec
-		{{ DBG_TRACE_GRAMMAR(only_all_class_spec, : only_class_name opt_partition_spec );
+                        $$ = ocs;
+                DBG_PRINT}}
+	;
 
-			PT_NODE *ocs = parser_new_node (this_parser, PT_SPEC);
+only_all_class_spec_with_server
```

Comment:
I think there is no syntactical consistency in meta_class_spec, only_all_class_spec_with_server, only_all_class_spec.

CLASS syntax is allowed in meta_class_spec.
However, only_all_class_spec_with_server does not allow ALL or EXCEPT syntax.

It seems that CLASS syntax is allowed in case of connecting from CUBRID to CUBRID.
However, because it is inconsistent, I think that CLASS syntax should not be supported either, or all grammars should be supported consistently.

Reply:
It's fixed.